### DISCUSSION TITLE
Solar Button Display

### DISCUSSION THREAD

**Discussion by NerdyD23** [aqualinkd/AqualinkD#176] [Q&A]
Enabling solar button in config.js does not make button visible on Safari mobile web browser.

Firefox and Chrome both show button on iOS mobile.

**Comment by sfeakes** [comment]
Aqualinkd uses very heavy caching, you need to clear the cache in safari for it to request the new config file.

**Reply by NerdyD23** [reply]
That worked! Thanks!
